odata-data-aggregation-ext/5 Vocabulary for Data Aggregation.md

```diff
@@ -365,3 +365,12 @@ results in
 :::
 
 Further examples for recursive hierarchies using transformations operating on the hierarchy structure are provided in [section ##AggregationinRecursiveHierarchies].
+
+## ##subsec Functions on Aggregated Entities
+
+Service-defined bound functions that serve as set transformations MAY be annotated with the term `AvailableOnAggregates` to indicate that they are applicable to aggregated entities under specific conditions:
+- The `RequiredProperties` collection lists all properties that must be available in the aggregated entities; otherwise, the annotated function will be inapplicable.
+
+::: example
+Example ##ex: assume the product is an implicit input for a function bound to a collection of `Sales`, then aggregating away the product makes this function inapplicable.
+:::
```

Comment:
Insert this, or deprecate term [`AvailableOnAggregates`](https://github.com/oasis-tcs/odata-vocabularies/blob/e69a1bf5b1d9ada497e5e8f271bdabe5f0589c19/vocabularies/Org.OData.Aggregation.V1.xml#L526-L533)